.travis.yml

```diff
@@ -71,6 +72,17 @@ jobs:
         - make verify-binapi
         - make integration-tests
 
+    - stage: E2E
```

Comment:
Let's make this a conditional stage and skip for pull_request and perhaps only run in master/dev branch if push and run always for cron scheduled ones.

Reply:
Agree, I just need to do some googling to find out how it is done ;)

Reply:
https://github.com/wip/action/blob/c1efe626e07157ba554333f9f5b2bce9b1f4726f/.travis.yml#L12-L15